### PR TITLE
Update common.js

### PR DESCRIPTION
**Pull request by timothywarner**
This pull request includes a small change to the `supportedDomains` array in the `common.js` file. The change removes several domains from the list of supported domains.

Changes to `supportedDomains` array:

* Removed `https://preview.portal.azure.com`
* Removed `https://make.powerapps.com`
* Removed `https://make.preview.powerapps.com`

### CODE REVIEW

**Review by timothywarner — COMMENTED** [review]
Wait are we for sure not using preview portals?

**Comment by timothywarner** [comment]
Close #2

File: common.js [hunk 1]
```diff
@@ -2,15 +2,12 @@ export const supportedDomains = [
     'https://portal.azure.com',
     'https://ms.portal.azure.com',
     'https://rc.portal.azure.com',
-    'https://preview.portal.azure.com',
```
Comment (alextaylor-the-dev) [inline common.js:5]: I think we actually use this. cc: **@timothywarner-org/pygoat-devs** where you at?

**Review by alextaylor-the-dev — DISMISSED** [review]
We need to discuss those URLs pls.

**Comment by timothywarner** [comment]
referring to the commit b679416

**Pull request merged**